inference-engine/tests/ie_test_utils/common_test_utils/test_common.hpp

```diff
@@ -6,13 +6,19 @@
 
 #include <gtest/gtest.h>
 
+#include <string>
+
 namespace CommonTestUtils {
 
 class TestsCommon : virtual public ::testing::Test {
 protected:
     TestsCommon();
 
     ~TestsCommon() override;
+
+    static std::string GetTimestamp();
+
+    static std::string GetTestName();
```

Comment:
`const`, not `static`

Reply:
Why this functions can't be `Static member functions` -> https://en.cppreference.com/w/cpp/language/static#Static_member_functions
They don't required object to call it.

Reply:
@iefode  can you answer why you don't want static here? I also think it should be a static function.

Reply:
I don't see the reason to use `static`.

Reply:
Done.